api_schemas/car_booking_schema.py

```diff
@@ -0,0 +1,23 @@
+from datetime import datetime
+from typing import Annotated
+
+from pydantic import StringConstraints
+from api_schemas.base_schema import BaseSchema
+from helpers.constants import MAX_CAR_DESC
+
+class CarRead(BaseSchema):
+    booking_id: int
+    user_id: int
+    description: str
+    start_time: datetime
```

Comment:
We have a `datetime_utc` which adds an extra nice validation. We should use that one on schema fields that need a timestamp